app/models/setting/provisioning.rb

```diff
@@ -40,6 +42,7 @@ def self.default_settings
     select = [{:name => _("Users"), :class => 'user', :scope => 'visible', :value_method => 'id_and_type', :text_method => 'login'},
               {:name => _("Usergroups"), :class => 'usergroup', :scope => 'visible', :value_method => 'id_and_type', :text_method => 'name'}]
 
+    transpiler_path = ['/var/lib/foreman/ct', '/usr/share/foreman/config/ct'].find { |p| File.directory?(p) } || File.join(Rails.root.to_s, "config", "ct")
```

Comment:
I don't think we should use paths in `/var/lib` in our coe. That's IMHO a packaging thing we can solve with a symlink.

Also note that in production `Rails.root` /usr/share/foreman` so it's slightly redundant. `Rails.root` is also a pathname so you can use `Rails.root.join('config', 'ct').to_s` for short.

Reply:
This is the standard practice in GNU but I will remove this to stick with our weird symlinking for easier dev setup.